crates/pint-query/src/main.rs

```diff
@@ -0,0 +1,152 @@
+use clap::Parser;
+use essential_rest_client::node_client::EssentialNodeClient;
+use essential_types::{convert::word_from_bytes, ContentAddress};
+use pint_abi::types::{ContractABI, TypeABI};
+use pint_manifest::ManifestFile;
+use std::{fs::read_dir, path::PathBuf, str::FromStr};
+
+#[derive(Parser, Debug)]
+#[command(version, about, long_about = None)]
+/// Tool to query state from an Essential node endpoint.
+struct Args {
+    /// The endpoint of node to bind to.
+    #[arg(long)]
+    node_address: String,
+    /// The contract address to query, encoded as hex.
+    #[arg(long)]
+    contract_address: ContentAddress,
+    /// The key name to query.
+    #[arg(long)]
+    key: Option<String>,
+    /// The path to the package manifest.
+    ///
+    /// If not provided, the current directory is checked and then each parent
+    /// recursively until a manifest is found.
+    #[arg(long)]
+    manifest_path: Option<PathBuf>,
+    /// The key to query, encoded as hex.
+    #[arg(long)]
+    key_hex: Option<Key>,
+}
+
+#[tokio::main]
+async fn main() {
+    let args = Args::parse();
+    if let Err(err) = run(args).await {
+        eprintln!("Command failed because: {}", err);
```

Comment:
```suggestion
        let bold = Style::new().bold();
        eprintln!("{}Error:{} {err:?}", bold.render(), bold.render_reset());
```

Something like this^ would be a little more consistent with pint. Also note the use of `{:?}` formatting for the error - this is because `anyhow::Error`'s `Display` implementation doesn't show the full error trace, but the `Debug` impl does (don't ask me why 🤷).